conjur/client.py

```diff
@@ -30,7 +30,7 @@ class ConfigException(Exception):
 
     This class is used to wrap a regular exception with a more-descriptive class name
 
-    *************** DEVELOPER NOTE ***************
+    `*************** DEVELOPER NOTE ***************`
```

Comment:
why this change

Reply:
don't know why this happened. good catch